domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/Formuegrunnlag.kt

```diff
@@ -236,3 +231,14 @@ sealed class KunneIkkeLageFormueVerdier {
 }
 
 fun List<Formuegrunnlag>.harEpsFormue() = this.any { it.epsFormue != null }
+
+/**
+ * Kaster exception dersom det finnes mer enn 1 element i listen,
+ * eller dersom listen er tom
+ */
+fun List<Formuegrunnlag>.firstOrThrowIfMultipleOrEmpty(): Formuegrunnlag {
+    if (this.size > 1) {
```

Comment:
```suggestion
    if (this.size != 1) {
```
Da får vi ikke 2 forskjellige exceptions fra samme funksjon.